lib/matplotlib/ticker.py

```diff
@@ -2503,7 +2490,6 @@ def __call__(self):
         return self.tick_values(vmin, vmax)
 
     def tick_values(self, vmin, vmax):
-        base = self._base
         linthresh = self._linthresh
```

Comment:
This is a bit weird to move, as they seem to be paired bits of information here.

Reply:
But not used as pairs.